composer.json

```
@@ -14,6 +14,10 @@
         {
             "type": "composer",
             "url": "https://updates.ez.no/ttl"
+        },
+        {
+            "type": "git",
+            "url": "https://github.com/ezsystems/ezplatform-personalization.git"
```

This configuration does not work on Ibexa Cloud
The following code works perfectly.
```suggestion
            "type": "vcs",
            "url": "https://github.com/ezsystems/ezplatform-personalization"
```

---

Should rather add it here: https://github.com/ezsystems/support-site/blob/master/updates-app/satis.json

Then we can just swap this repo to start to use updates.ibexa.co for master in order to start to actively test it.
See #176

---

Solving this in link below for updates.ibexa.co

---

I commented without looking at the discussion first. This PR was shared with me 2 days ago so I can install it on p.sh. 
I agree it should be added to Satis. All good for me.